packages/core/admin/ee/admin/content-manager/pages/ListView/ReviewWorkflowsColumn/ReviewWorkflowsAssigneeEE.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { Typography } from '@strapi/design-system';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+
+import getTrad from '../../../../../../admin/src/content-manager/utils/getTrad';
+
+export function ReviewWorkflowsAssigneeEE({ firstname, lastname }) {
+  const { formatMessage } = useIntl();
+
+  if (!firstname && !lastname) {
+    return <Typography textColor="neutral800">-</Typography>;
+  }
+
+  return (
+    <Typography textColor="neutral800">
+      {formatMessage(
+        {
+          id: getTrad(`containers.ListPage.reviewWorkflows.assignee`),
+          defaultMessage: '{firstname} {lastname}',
+        },
+        { firstname, lastname }
+      )}
```

Comment:
Could you add a TODO here, to align this with https://github.com/strapi/strapi/pull/17043/files#diff-19fead44beaa2b6aa3c5ce92728dfa068d4f10c7225bc53f9d6c2df4a765bd59R20-R21 ?

Also: this should probably start to use the util the other PR introduces.

Reply:
I think I see what you mean. We should use `getDisplayName` once the other PR is ready right? does that TODO cover it?

Reply:
Yes I think this is fine. Thank you!